it/src/test/resources/com/linecorp/centraldogma/it/mirror/git/test.pem

```diff
@@ -0,0 +1,20 @@
+-----BEGIN EC PRIVATE KEY-----
```

Comment:
Question) Just to make sure, is this a read-only deploy key?

Reply:
Yes, it's read-only.